app/lib/frontend/templates/_utils.dart

```diff
@@ -8,32 +8,34 @@ import 'package:path/path.dart' as p;
 
 import '../../package/model_properties.dart' show FileObject;
 import '../../shared/markdown.dart';
+import '../dom/dom.dart' as d;
 
 const HtmlEscape htmlAttrEscape = HtmlEscape(HtmlEscapeMode.attribute);
 
 /// Renders a file content (e.g. markdown, dart source file) into HTML.
-String renderFile(
+d.Node renderFile(
   FileObject file, {
   String? baseUrl,
   bool isChangelog = false,
 }) {
   final filename = file.filename;
   final content = file.text;
   if (_isMarkdownFile(filename)) {
-    return markdownToHtml(content,
+    return d.unsafeRawHtml(
+      markdownToHtml(
+        content,
         baseUrl: baseUrl,
         baseDir: p.dirname(filename),
-        isChangelog: isChangelog)!;
+        isChangelog: isChangelog,
+      )!,
```

Comment:
Can't we create a `d.safeHtml` which calls `sanitizeHtml`, then we don't have to call `sanitizeHtml` in `markdownToHtml`.

Or maybe we refactor these bits later... Just saying, we can slowly start to remove some of the calls to `d.unsafeRawHtml`, if we build things like `d.safeHtml`...

Reply:
Yeah, we should certainly refactor some parts. We also have `unsafeRawHtml` in the announcement banner and in the consent description rendering. I think those are similar use-cases where we could use the sanitization on the HTML text.